### PR TITLE
Refactor flexcontainer reuse

### PR DESCRIPTION
**Pull request by atpradical**
flexcontainer reused for exist components and pages to provide better user experience.

### CODE REVIEW

File: src/components/ui/primitives/radio/radio.tsx [hunk 1]
```diff
@@ -21,22 +22,21 @@ export const Radio = forwardRef<RadioGroupRef, RadioProps>((props, ref) => {
...
-    wrapper: clsx(s.wrapper),
   }
 
   const defaultOption = options.find(i => i.defaultValue)?.value || options[0].value
 
   return (
     <RadixRadio.Root className={cn.root} defaultValue={defaultOption} ref={ref} {...rest}>
       {options.map(i => (
-        <div className={cn.wrapper} key={i.id}>
+        <FlexContainer key={i.id}>
```
Comment (EleonoraPavlova) [inline src/components/ui/primitives/radio/radio.tsx:32]: cool

File: src/components/ui/primitives/tab-switcher/tab-switcher.tsx [hunk 2]
```diff
@@ -23,8 +23,7 @@ export const TabSwitcher = forwardRef<TabSwitcherRef, Props>(
   ({ className, defaultValue, label, tabs, ...rest }, ref) => {
     const cn = {
       label: clsx(s.label),
-      list: clsx(s.tabsList),
-      root: clsx(s.tabsRoot, className),
+      root: clsx(className),
```
Comment (EleonoraPavlova) [inline src/components/ui/primitives/tab-switcher/tab-switcher.tsx:26]: поддерживаю

**Review by EleonoraPavlova — APPROVED** [review]
good job! looks attractively

**Review by M4k4v3ll1 — APPROVED** [review]
Awesome!

**Pull request merged**